src/chart_types/xy_chart/store/chart_state.ts

```diff
@@ -134,7 +134,7 @@ export const isDuplicateAxis = (
     ) {
       const spec = specMap.get(axisId);
 
-      if (spec && spec.position === position && ((!title && !spec.title) || title === spec.title)) {
+      if (spec && spec.position === position && title === spec.title) {
```

Comment:
it's a double equal here if we want to mark as duplicate if one title is null and the other undefined (that in our case we can threat as no value)
```suggestion
      if (spec && spec.position === position && title == spec.title) {
```

Reply:
The value cannot be `null` based on the types correct?

Nonetheless, I also don't like double equals in javascript because it is loose equality, and generally a bad practice. It also can cause unwanted conversions producing inaccurate results. I'd like to enforce this throughout the code eventually.

I think the explicit equality adding twice the code to clearly indicate the comparison is worth it.

Reply:
yep that value cannot be null, but I was thinking on the case the chart used in a JS world (people can do strange things there).
You are right about the loose equality, I use that only in rare case checking when checking for null and undefined, in that case you are right, we can get some strange results. So please ignore my comment here

Reply:
To your point about people using this library in javaScript and not minding the types.

I 100% agree, but I think if we want to address that there are numerous places where we assume values to be provided/required. I think the `Theme` is a perfect example where we assume options are provided on the type were in js they may not.